### PR TITLE
feat(build): Run tests with GitHub actions

### PR DESCRIPTION
**Pull request by YonatanKra**
Solves #53

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/dist
+/idea
+/node_modules
+/@types
+/testingP
+*-lock
```
Comment (mhagmajer) [inline .prettierignore:6]: seem like we don't have these at the top level

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/dist
+/idea
```
Comment (mhagmajer) [inline .prettierignore:2]: seem like we don't have these at the top level

Comment (YonatanKra) [reply .prettierignore]: Correct :)

File: .prettierignore (hunk 1 above)
Comment (YonatanKra) [reply .prettierignore]: I've removed testingP and *-lock.
I do see @types when developing locally, so I guess it should also be ignored

**Pull request merged**